tool/jt.rb

```diff
@@ -542,6 +542,7 @@ def add_default_dynamic_imports
     env_lines = (File.exist?(env_path) ? File.read(env_path) : '').lines.map(&:chomp)
     dynamic_import_line = env_lines.find { |l| l.include? 'DEFAULT_DYNAMIC_IMPORTS' } || (env_lines[env_lines.size] = 'DEFAULT_DYNAMIC_IMPORTS=')
     dynamic_imports = dynamic_import_line.split('=', 2).last.split(',')
+    return if dynamic_imports.include?('/tools')
```

Comment:
With the early return, you can eliminate the conditional part of the following line.

Reply:
Done.